### PR TITLE
Add group stats

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -741,6 +741,41 @@ curl "https://airbrake.io/api/v4/groups?key=USER_KEY"
...
+
+`GET https://airbrake.io/api/v4/groups/GROUP_ID/stats?key=USER_KEY`
+
+### Query parameters
+
+Parameter | Default | Description
+--------- | ------- | -----------
+period | 1h | Aggregates results for the period. Supported periods are: 1m, 5m, 15m, 30m, 3h, 6h, 12h, and 24h.
+time__since | | Returns results since the time.
+time__before | | Returns results before the time.
```
Comment (kyrylo) [inline source/index.html.md]: We should probably specify if this is an inclusive or exclusive range.

File: source/index.html.md [hunk 2]
```diff
@@ -741,6 +741,41 @@ curl "https://airbrake.io/api/v4/groups?key=USER_KEY"
...
+```shell
+curl "https://airbrake.io/api/v4/groups/GROUP_ID/stats?key=USER_KEY"
+```
+
+```json
+{
+  "projectId": PROJECT_ID,
+  "groupId": GROUP_ID,
+  "accepted": [904, 2013],
+  "limited": [6784, 4245],
```
Comment (kyrylo) [inline source/index.html.md]: What is this?

Comment (vmihailenco) [reply source/index.html.md]: Number of notices dropped by backend, because group is too big.

File: source/index.html.md (hunk 1 above)
Comment (vmihailenco) [reply source/index.html.md]: For me it sounds like:
- since is inclusive
- after is exclusive
- before is exclusive

Comment (kyrylo) [reply source/index.html.md]: Why not to put it in the actual docs?

File: source/index.html.md (hunk 2 above)
Comment (kyrylo) [reply source/index.html.md]: Do you think it needs an explanation?

**Pull request merged**